source/background.ts

```diff
@@ -42,6 +42,13 @@ browser.runtime.onMessage.addListener(async message => {
 	}
 });
 
+// Give the browserAction a reason to exist other than "Enable RGH on this domain"
+browser.browserAction.onClicked.addListener(() => {
+	browser.tabs.create({
+		url: 'https://github.com'
+	});
+});
```

Comment:
Wouldn't it be more useful if the button opened the settings? `chrome://extensions/?options=lcdgbjnbapalaoeehpdlmkbbengbechl`

Reply:
Maybe. The settings are already available with a right click though. By opening `github.com` it could become a shortcut to it, instead of a rarely-used one-less-click shortcut to Options.

Reply:
I don't find the button useful regardless, so I don't feel strongly about this. `¯\_(ツ)_/¯`

Reply:
Same, but I might keep it visible if it’s a shortcut to GitHub